package.json

```diff
@@ -2,6 +2,9 @@
   "name": "app",
   "version": "0.1.0",
   "private": true,
+  "prisma": {
```

Comment:
I think you forgot to change the `app` service on the docker-compose to run `prisma db seed`.

Were you able to make it auto-run previous to the `prisma db migrate dev`?